### DISCUSSION TITLE
Fordpass now supports displaying Charge rate (KW) -- Does the SDK/API?

### DISCUSSION THREAD

**Discussion by pc500** [ianjwhite99/sync-connect-sdk#75] [Q&A]
The new app now shows charge rate for the current session.  Is this the result of an API change that the SDK needs to support?  Or it just something computational the app did?
![image](https://user-images.githubusercontent.com/33823824/227620230-f23934c0-13f5-49c1-9cba-5ac45e65313f.png)

**Comment by ianjwhite99** [comment]
Not sure without looking at the API more closely for a EV. I unfortunately don't have an EV, so I am unable to see these API requests. If someone in the community would be able to provide API endpoints and data it would be possible to implement some sort of solution for the SDK.
